src/main/java/seedu/address/model/person/Person.java

```diff
@@ -176,7 +221,7 @@ public boolean isSamePerson(Person otherPerson) {
         return otherPerson != null && otherPerson.getNric().equals(getNric())
                 && otherPerson.getName().equals(getName())
                 && (otherPerson.getPhone().equals(getPhone())
-                        || otherPerson.getEmail().equals(getEmail()));
+                || otherPerson.getEmail().equals(getEmail()));
```

Comment:
Why was the formatting changed here?
Please do not change other people's formatting on your own.